core/src/main/res/values/strings.xml

```diff
@@ -863,6 +865,13 @@
     <string name="notification_channel_auto_download">Auto Downloads</string>
     <string name="notification_channel_episode_auto_download">Shown when episodes have been automatically downloaded.</string>
 
+    <string name="notification_channel_new_episode">New Episode</string>
```

Comment:
Could you please remove the newline before this and move `new_episode_message` further up to `episode_notification_summary`? I think the section here is only about the notification channels themselves.